src/ArticleCard.jsx

```diff
@@ -0,0 +1,26 @@
+import React, { useEffect, useState } from 'react';
+import { get } from './utils/api';
+import './index.css';
+
+function ArticleCard() {
+    const [articles, setArticles] = useState([]);
+
+    useEffect(() => {
+        get('/articles')
+        .then((data) => setArticles(data.articles))
+        .catch((err) => console.error('Error fetching articles:', err));
```

Comment:
Rather than console logging this error, it may be good if it's displayed for the user?